docs/guides/create-webshop.md

```diff
@@ -21,11 +21,11 @@ Get the latest version of the [Orchard Core Commerce NuGet](https://www.nuget.or
 
 ### Step 2 — Enable necessary features
 
-Certain Orchard Core Commerce features, as well as some prerequisite stock Orchard Core features, need to be enabled for the necessary functionality. This can be done under _Configuration > Features_. These features include the following:
+Certain Orchard Core Commerce features, as well as some prerequisite stock Orchard Core features, need to be enabled for the necessary functionality. This can be done under _Configuration > Features_, but if you used the _Orchard Core Commerce - Development_ setup recipe then they are already enabled. These features include the following:
 
 - **Orchard Core Commerce - Core**: Registers core components of Commerce features.
 - **Orchard Core Commerce - Payment** Provides the basics for online payment.
-- **Orchard Core Commerce - Payment - Stripe**: Provides online payment implementation using Stripe.
+- **Orchard Core Commerce - Payment - Exactly**: Provides online payment implementation using Exactly.
```

Comment:
```suggestion
- **Orchard Core Commerce - Payment - Exactly**: Provides online payment implementation using Exactly®.
```

Reply:
That's the feature name, where I'd prefer to not use special characters.